dist/style.css

```diff
@@ -1,61 +1,66 @@
 body {
-  font-family: Arial, sans-serif;
-  background-color: lightgray;
-  padding: 100px;
+    font-family: Arial, sans-serif;
+    background-color: lightgray;
+    padding: 100px;
 }
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍 Just like I have made my logo which you can see in the image above ⬆️